docker/standalone/cpu/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:14.04
+#MAINTAINER fixme@example.com
```

Comment:
Could you set this to caffe-maint@googlegroups.com ? We've set up this forwarding list to handle packaging and distribution maintenance. Thanks, and sorry for the wait while that was set up.